### PR TITLE
Feat/amy board

### PR DESCRIPTION
**Pull request by amspears007**
Started Cell class and valid coordinates and and valid placements methods and test

### CODE REVIEW

File: lib/board.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+class Board
+  attr_reader :cells
+
+  def initialize
```
Comment (amspears007) [inline lib/board.rb:4]: Created Hash of cells for board

File: lib/board.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      "C3" => Cell.new('C3'),
+      "C4" => Cell.new('C4'),
+      "D1" => Cell.new('D1'),
+      "D2" => Cell.new('D2'),
+      "D3" => Cell.new('D3'),
+      "D4" => Cell.new('D4')
+     }
+  end
+
+  def valid_coordinate?(coordinate)
```
Comment (amspears007) [inline lib/board.rb:25]: created valid_coordinate? method

File: lib/board.rb [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    cells.keys.include?(coordinate)
+  end
+
+  def valid_placement?(ship, coordinates)
```
Comment (amspears007) [inline lib/board.rb:29]: started valid_placement method

**Review by angelbyun — APPROVED** [review]
Great start on the valid_placement? method!

**Pull request merged**